DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/README.md

```diff
@@ -178,9 +178,7 @@ PASSED
 |`mvdr_complex.hpp`              | Definition of ComplexType, used throughout this design
 |`MVDR.hpp`                      | Function to launch all MVDR kernels and define the pipes that connect them together
 |`ParallelCopyArray.hpp`         | Defines the ParallelCopyArray class, an array that supports unrolled copy / assign operations
-|`pipe_array.hpp`                | Header file containing the definition of an array of pipes
-|`pipe_array_internal.hpp`       | Helper for pipe_array.hpp
-|`PipeDuplicator.hpp`            | Defines the PipeDuplicator class, creates multiple copies of a pipe for fan-out
+|`pipe_utils.hpp`                | Header file containing the definition of an array of pipes and a pipe duplicator
```

Comment:
Should we mention that this exists in a different directory. Maybe the directory structure should be setup such that the include line looks like:

`#include "hldutils/pipe_utils.hpp"`

Reply:
I can definitely update the README text to clarify where the header resides, but I'm not sure if we can setup the include structure in that way. @mdbtucker do you know?

Reply:
The directory path to the file will be .../DPC++FPGA/include_pipe_utils.hpp.  I don't think we want to add another sub-directory under 'include' just for the sake of making it more obvious that this file resides elsewhere.  I think mentioning the location in both the README and in a comment in the code where the file is #included is adequate.

Reply:
I won't fight this much. But not that it is common practice. E.g., https://github.com/intel/llvm/tree/sycl/sycl/include/CL/sycl

Notice the `CL/sycl` directory inside `include`

Reply:
That seems redundant to me - 'sycl' is in the path three times?!?  I'm not sure that's a pattern we should emulate.

That said, if you think it's helpful, I'm open to another directory, but I don't think 'hldutils' is a good choice of name.  What about DPC++FPGA/include/sycl_fpga?  I don't really love it, but I'm not 100% opposed either.

Reply:
Again. I am only suggesting this because it is a widely accepted convention.

https://github.com/boostorg/accumulators/tree/14c13370602fe86d134a948943958cab0921ce9c/include/boost/accumulators

In terms of names; we could send an email to select ppl and get their feedback.

Reply:
For the initial version of this directory, I'm going to leave it as-is based on the preference shown for it in previous conversations. We can refactor this later if needed.